demo/pages/menu.py

```diff
@@ -36,7 +56,7 @@
     mui.Grid(
```

Comment:
This page only has 2 examples, but the grid is done for 3 examples. There is a lot of white space on the right. I suggest we do a grid of two elements, so we can see the full code instead of having extra white space